sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -136,4 +147,53 @@ export abstract class ECertCalculationProcess {
       }
     }
   }
+
+  /**
+   * Creates disbursement blocked notifications for student and ministry.
+   * @param studentId student id associated with the blocked disbursement.
+   * @param applicationNumber application number associated with the blocked disbursement.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  private async createDisbursementBlockedNotifications(
+    studentId: number,
+    applicationNumber: string,
+    entityManager: EntityManager,
+  ) {
+    const student = await entityManager.getRepository(Student).findOne({
+      select: {
+        id: true,
+        birthDate: true,
+        user: { firstName: true, lastName: true, email: true },
+      },
+      relations: { user: true },
+      where: { id: studentId },
+    });
+    const studentNotification = {
```

Comment:
I would suggest to have the constants typed. 
![image](https://github.com/bcgov/SIMS/assets/54600590/716c9bbc-e220-4998-85ab-8aa66af884e1)